core/modules/ccontrol/UserQueryFactory.cc

```diff
@@ -40,18 +40,23 @@
 #include "ccontrol/UserQueryDrop.h"
 #include "ccontrol/UserQueryFlushChunksCache.h"
 #include "ccontrol/UserQueryInvalid.h"
+#include "ccontrol/UserQueryProcessList.h"
```

Comment:
My smart editor folds copyright comment. I wish we got rid of all of that already.